src/orquestra/sdk/mlflow/_connection_utils.py

```diff
@@ -0,0 +1,36 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+
+"""Utilities for communicating with mlflow."""
+
+import os
+from pathlib import Path
+
+from orquestra.sdk._base._services import ORQUESTRA_BASE_PATH
+
+DEFAULT_TEMP_ARTIFACTS_DIR: Path = ORQUESTRA_BASE_PATH / "mlflow" / "artifacts"
+
+
+def get_temp_artifacts_dir() -> Path:
+    """
+    Return a path to a temp directory that can be used to temporarily store artifacts.
+
+    Uploading artifacts to MLFlow requires them to be written locally first. Finding an
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
```suggestion
    Uploading artifacts to MLflow requires them to be written locally first. Finding an
```